### ISSUE TITLE
oxlint/lsp: Quick fix lint disable action inserts invalid comment in JSX

### ISSUE THREAD

**Issue by cellison-figma** [oxc-project/oxc#24570]
### Confirm that this is a Bug for the editor extension, not the CLI

- [x] This bug does not happen in the Oxlint/Oxfmt CLI
- [x] After restarting the editor, the bug still appears

### Versions used

```
VS Code extension: v1.58.0
oxlint: v1.72.0
oxfmt: N/A
VS Code version: 1.109.5
Operating System and Version: MacOS Tahoe 26.5.2 (25F84)
Node Version: v 22.21.1
Node Manager: mise
```

### What happened?

For a JSX rule violation (e.g. for `jsx-a11y/interactive-supports-focus`), the Quick Fix "disable rule for line" action inserts a standard comment (`// oxlint-disable ...`) rather than a brace-escaped comment (`{/* oxlint disable ... */}`. This results in invalid JSX and requires an additional edit to correct.

### Reproduction repo

With this file, using the Oxc VSCode extension to insert a disable statement above the `<div>` will insert an invalid comment in JSX.

```tsx
import * as React from 'react'

// oxlint-enable jsx-a11y/interactive-supports-focus
export function Redirect() {
  return (
    <>
      <p>Another child to justify fragment</p>
      <div onClick={() => history.redirect('/')} role="link" />
    </>
  )
}
```

### Output channel log

N/A
